test/documentation.spec.ts

```diff
@@ -19,84 +19,100 @@ describe('documentation', () => {
   });
 
   describe('website-documentation', () => {
-    describe('configuration-options', () => {
-      const doc = fs.readFileSync(
-        'docs/usage/configuration-options.md',
-        'utf8',
-      );
-
-      const headers = doc
-        .match(/\n## (.*?)\n/g)
-        ?.map((match) => match.substring(4, match.length - 1));
-
-      const expectedOptions = options
-        .filter((option) => !option.globalOnly)
-        .filter((option) => !option.parents)
-        .filter((option) => !option.autogenerated)
-        .map((option) => option.name)
-        .sort();
+    describe('docs/usage/configuration-options.md', () => {
+      function getConfigHeaders(file: string): string[] {
+        const content = fs.readFileSync(`docs/usage/${file}`, 'utf8');
```

Comment:
I would prefer async io

Reply:
My objective was to make it straightforward: I see the Jest failed test, I know which Markdown to search. The `await` instructions will only create more noise here.

The performance/efficiency is secondary concern for this particular test case, because it is the separate CI job anyway, and runs/fails very fast.